Importing ESlint reports shouldn't spam output with errors if no report found

Issue:

I have a solution (MSBuild) that builds 62 projects. Only 3 of those projects contain JavaScript / TypeScript. I want to import Eslint reports into SonarQube, to have a unified way of code quality reporting. However I'm blocked in doing so as SonarScanner will report 59 errors while scanning. It tries to import eslint's report for all projects, even though most projects don't emit eslint reports. This produces a lot of errors, which makes it hard to find any real errors within the pile of errors generated by SonarScanner.

Expected result: don't print errors if no report is found

Actual result: output is littered with "errors"

My version:
```
SonarScanner for MSBuild 4.9
INFO: SonarScanner 4.3.0.2102
```

Build: MSBuild on Azure Devops

Configuration:
```
sonar.eslint.reportPaths=eslint.json
```

Logs:

![image](https://user-images.githubusercontent.com/235882/86019773-50c80c80-ba27-11ea-9aff-e87a139a597d.png)

```
2020-06-29T14:17:47.2683996Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:47.2685764Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:47.2686612Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:47.2696101Z ##[error]at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:69)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:68)
2020-06-29T14:17:47.2701444Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:47.2702066Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:47.2702736Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:47.2703531Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)

2020-06-29T14:17:47.8092726Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
2020-06-29T14:17:47.8102925Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:47.8105366Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:47.8106614Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:47.8107654Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:47.8108740Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:47.8109965Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:47.8111978Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:47.8113258Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:47.8114502Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:47.8122574Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:47.8123946Z 	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)

2020-06-29T14:17:48.0631350Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:69)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:68)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:122)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:108)
	at org.sonar.scanner.scan.ProjectScanContainer.scan(ProjectScanContainer.java:418)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:414)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:411)
	at org.sonar.scanner.scan.ProjectScanContainer.doAfterStart(ProjectScanContainer.java:372)

2020-06-29T14:17:48.2414789Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:48.2422728Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:48.2431455Z ##[error]java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)
2020-06-29T14:17:48.2437396Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:48.2438629Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:48.2439646Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:48.2441025Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:48.2442271Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:48.2443686Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:48.2444946Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:48.2446306Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)

2020-06-29T14:17:48.4933718Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:48.4938964Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:48.4943125Z ##[error]java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:48.4945835Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:48.4946946Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:48.4947880Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:48.4948913Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:48.4955870Z ##[error]at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:48.4958562Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:48.4959989Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:48.4961618Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:48.4964970Z ##[error]at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:48.4967367Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:48.4968730Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:48.4975322Z ##[error]at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)

2020-06-29T14:17:48.7285508Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:48.7293393Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:48.7294775Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:48.7295848Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:48.7296882Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:48.7300174Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:48.7301349Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:48.7322542Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:48.7323846Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:48.7325031Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:48.7326374Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:48.7346898Z ##[error]at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)

2020-06-29T14:17:48.9451546Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:48.9454882Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:48.9456090Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:48.9458209Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:48.9459219Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:48.9460791Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:48.9462013Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:48.9463159Z INFO: Sensor Import of ESLint issues [javascript] (done) | time=2ms
2020-06-29T14:17:48.9464907Z INFO: Sensor ThymeLeaf template sensor [securityjavafrontend]
2020-06-29T14:17:48.9491704Z ##[error]at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:69)

2020-06-29T14:17:49.3467769Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:69)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:68)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:122)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:108)
	at org.sonar.scanner.scan.ProjectScanContainer.scan(ProjectScanContainer.java:418)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:414)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:411)
	at org.sonar.scanner.scan.ProjectScanContainer.doAfterStart(ProjectScanContainer.java:372)

2020-06-29T14:17:49.5618380Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:49.5620064Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:49.5629096Z ##[error]java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:49.5635213Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:49.5636369Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:49.5639061Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:49.5640122Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:49.5641680Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:49.5642982Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:49.5644097Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:49.5645198Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:49.5649714Z ##[error]at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)

2020-06-29T14:17:49.7736994Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:69)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:51)
	at org.sonar.scanner.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:68)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:122)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:108)
	at org.sonar.scanner.scan.ProjectScanContainer.scan(ProjectScanContainer.java:418)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:414)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:411)
	at org.sonar.scanner.scan.ProjectScanContainer.doAfterStart(ProjectScanContainer.java:372)

2020-06-29T14:17:49.9505679Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:49.9510963Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:49.9512333Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:49.9518100Z ##[error]at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:49.9520521Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:49.9522408Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:49.9523511Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:49.9524806Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:49.9528048Z ##[error]at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:49.9531058Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:49.9532449Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:49.9533666Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:49.9536242Z ##[error]at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
2020-06-29T14:17:49.9538922Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:49.9540234Z 	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)

2020-06-29T14:17:50.0825503Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:50.0827423Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:50.0832549Z ##[error]java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:50.0835987Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:50.0843499Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:50.0844558Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:50.0849197Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:50.0853462Z ##[error]at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:50.0856670Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:50.0858532Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:50.0859836Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:50.0867547Z ##[error]at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:50.0889989Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:50.0891658Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:50.0894191Z ##[error]at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:78)

2020-06-29T14:17:50.2850854Z ##[error]ERROR: No issues information will be saved as the report file can't be read.
java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:50.2853275Z ERROR: No issues information will be saved as the report file can't be read.
2020-06-29T14:17:50.2854425Z java.io.FileNotFoundException: D:\a\1\s\Source\SOME_PROJECT\eslint.json (The system cannot find the file specified)
2020-06-29T14:17:50.2856885Z ##[error]at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:50.2858763Z 	at java.base/java.io.FileInputStream.open0(Native Method)
2020-06-29T14:17:50.2859680Z 	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
2020-06-29T14:17:50.2860925Z 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
2020-06-29T14:17:50.2864395Z ##[error]at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:50.2866857Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.importReport(EslintReportSensor.java:61)
2020-06-29T14:17:50.2868511Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.lambda$execute$1(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:50.2869707Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
2020-06-29T14:17:50.2873152Z ##[error]at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)
2020-06-29T14:17:50.2877389Z 	at org.sonar.plugins.javascript.external.AbstractExternalIssuesSensor.execute(AbstractExternalIssuesSensor.java:54)
2020-06-29T14:17:50.2878676Z 	at org.sonar.plugins.javascript.external.EslintReportSensor.execute(EslintReportSensor.java:43)
2020-06-29T14:17:50.2879849Z 	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:34)

(and on it goes for 1360 lines...)
```

--- a/sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/external/EslintReportSensor.java
+++ b/sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/external/EslintReportSensor.java
@@ -78,7 +78,7 @@ public class EslintReportSensor extends AbstractExternalIssuesSensor {
         }
       }
     } catch (IOException | JsonSyntaxException e) {
-      LOG.error(FILE_EXCEPTION_MESSAGE, e);
+      LOG.warn(FILE_EXCEPTION_MESSAGE, e);
     }
   }
 

--- a/sonar-plugin/sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/external/EslintReportSensorTest.java
+++ b/sonar-plugin/sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/external/EslintReportSensorTest.java
@@ -116,7 +116,7 @@ class EslintReportSensorTest {
     Collection<ExternalIssue> externalIssues = context.allExternalIssues();
     assertThat(externalIssues).hasSize(0);
 
-    assertThat(logTester.logs(LoggerLevel.ERROR))
+    assertThat(logTester.logs(LoggerLevel.WARN))
       .contains("No issues information will be saved as the report file can't be read.");
   }
 
@@ -128,7 +128,7 @@ class EslintReportSensorTest {
     Collection<ExternalIssue> externalIssues = context.allExternalIssues();
     assertThat(externalIssues).hasSize(0);
 
-    assertThat(logTester.logs(LoggerLevel.ERROR))
+    assertThat(logTester.logs(LoggerLevel.WARN))
       .contains("No issues information will be saved as the report file can't be read.");
   }
 

